examples/basic/verifying_md.py

```diff
@@ -0,0 +1,203 @@
+import desbordante
+import pandas as pd
+
+from typing import TypedDict
+from desbordante.md_verifier.similarity_measures import (
+    SimilarityMeasure,
+    EuclideanSimilarity,
+    LevenshteinSimilarity,
+)
+
+GREEN_CODE = "\033[1;42m"
+RED_CODE = "\033[1;41m"
+BLUE_CODE = "\033[1;46m"
+DEFAULT_COLOR_CODE = "\033[1;49m"
+
+
+class MDParams(TypedDict):
+    lhs_indices: list[int]
+    rhs_indices: list[int]
+    lhs_decision_boundaries: list[float]
+    rhs_decision_boundaries: list[float]
+    lhs_similarity_measures: list[SimilarityMeasure]
+    rhs_similarity_measures: list[SimilarityMeasure]
+
+
+def print_results(verifier):
+    if verifier.md_holds():
+        print(GREEN_CODE, "MD holds", DEFAULT_COLOR_CODE, "\n")
+        return
+
+    highlights = verifier.get_highlights()
+    print(RED_CODE, "MFD does not hold due to the following items:", DEFAULT_COLOR_CODE)
+    for highlight in highlights:
+        print(
+            f"Rows {highlight.rows[0]} and {highlight.rows[1]} violate MD in column {highlight.column}: "
+            f'"{highlight.first_value}" and "{highlight.second_value}" have similarity {highlight.similarity} '
+            f"with decision boundary {highlight.decision_boundary}"
+        )
+    print(
+        f"Desbordante suggests to use following right-hand side decision boundaries: {verifier.get_rhs_suggestions()}\n"
+    )
+
+
+def check_md(table_path: str, params: MDParams):
+    algo = desbordante.md_verifier.algorithms.Default()
+    algo.load_data(table=(table_path, ",", True))
+
+    algo.execute(**params)
+    print_results(algo)
+
+
+def drunk_animals_example():
```

Comment:
I just chose a name that's kind of funny in the original PR, but use the current one